### ISSUE TITLE
Funny fonts after default state

### ISSUE THREAD

**Issue by kentron** [digitalfiz/language-lsl#20]
After upgrading Atom to 1.57.0, I found ![Atom LSL mode funny chars Screenshot 2021-09-08 - 11 45 53](https://user-images.githubusercontent.com/881275/132531843-ad064fcf-4e1f-4939-a9ce-0b2c2cb87c5f.png) all my LSL code change font and background after default state, since it seems that 'default' is not being recognized as part of language LSL. when typing other word, the chars come back to normal. Changing the file type to C in Atom, renders the file correctly too.

**Comment by kentron** [comment]
HI, I got some spare time after new year vacations and I tweaked file language-lsl.cson inside snippets, adding a default.

I am not sure if my syntax is right, but my files look fine now.

Can someone point if that is the right solution?

Thank you in advance,
![language-lsl cson hack](https://user-images.githubusercontent.com/881275/149452067-8d13e8b0-a76c-4aab-9f65-88207e780706.png)
K

**Comment by digitalfiz** [comment]
Yeah that looks correct. I will release a new version this evening when I get off of work. I think there are a few other things in the issues section I can address also.
